### DISCUSSION TITLE
Volume Flow

### DISCUSSION THREAD

**Discussion by marcus198006** [signalkraft/mypyllant-component#82] [Ideas]
Hello, is it possible to get the current volume flow of the water circuit? The hydraulic station shows me the value, I wonder if it is possible to get it as an entity?

Thanks

**Comment by signalkraft** [comment]
Is that possible in the myVAILLANT app? If not, it's unlikely that it can be added.

**Reply by marcus198006** [reply]
I can confirm that it is not possible in the myVAILLANT App.

**Comment by rmalbrecht** [comment]
Sensocomfort get's it via eBus from the Arotherm, but I have never seen it in the app.

![Bildschirmfoto 2024-02-07 um 22 25 33](https://github.com/signalkraft/mypyllant-component/assets/49707000/7a7c6b4d-0f89-487e-9d7f-561e1488baa3)

**Reply by marcus198006** [reply]
Hi Robert, ok danke, aber wie hast du den Volumenstrom vom eBUS nach HA bekommen (also ich vermute, dass du das hast)?

**Reply by rmalbrecht** [reply]
Hi,
es gibt verschiedene eBus-Adapter, mit denen man an die interne Kommunikation zwischen den Vaillant Komponenten rankommt.
https://romal.de/?s=ebus
https://github.com/john30/ebusd-configuration
https://github.com/rmalbrecht/ebusd-configuration
Signalkrafts Addon ist viel "einfacher" (man muss die Heizung nicht aufschrauben) und stabiler. Dafür ist die interne Kommunikation informativer.
cu romal

**Reply by marcus198006** [reply]
Nice! Danke

**Reply by rmalbrecht** [reply]
eBus is a really deep rabbit hole. And eBus access can properly confuse (and maybe even damage) your heating.

**Reply by marcus198006** [reply]
Not promising and that's my first impression by reading your blok. I will try to understand it and if I conclude that it is too risky, i want go further down that road. 
I also requested Vaillant to get dQ/dt into the API. First respond was that it is only available for professionals with professional software. Not proper reason to me. They check it. Let's see.

**Comment by Presl1983** [comment]
is volume flow the same as volume temperatur?
in the old vaillant integration it was possible to get the volume temperatur. Is it possible to get in via myvaillant - i had to upgrade yesterday.

**Comment by rmalbrecht** [comment]
What is volume temperature? Volume flow is the amount of liters the pump is pushing through the underfloor heating pipes.

**Reply by Presl1983** [reply]
Vorlauftemperatur in German 🙈

**Comment by rmalbrecht** [comment]
https://signalkraft.com/mypyllant-component/2-entities/ 

lists the entities. Maybe you are looking for 

** Home Circuit 0 Current Flow Temperature**

For my installation, this is the supply temperature. For heating it's around 22°. For hot water, it's changing a lot.
